server/app/views/v1/containers/_container.json.jbuilder

```diff
@@ -41,3 +41,7 @@ else
   json.overlay_cidr nil
 end
 json.network_settings container.network_settings
+if container.health_status
+  json.health_status container.health_status
+  json.health_status_at container.health_status_at
```

Comment:
Ok, got it (need more coffee). I misread that in a way that json has `health_status` object.. somehow it sounds more natural to me. Is it a bad idea? (internal db model does not have to reflect directly to json api)